databaseserviceexample.gitlab-ci.yml

```diff
@@ -0,0 +1,45 @@
+include: 
+  - https://raw.githubusercontent.com/Cimpress-MCP/gitlab-continuous-integration/master/belt-node-complexity-analysis-ci.yml
+  - https://raw.githubusercontent.com/Cimpress-MCP/gitlab-continuous-integration/master/deploy/deploycomponents.yaml
+  - https://raw.githubusercontent.com/Cimpress-MCP/gitlab-continuous-integration/master/deploy/beanstalkdeploy.yaml
+  - https://raw.githubusercontent.com/Cimpress-MCP/gitlab-continuous-integration/master/setup/nodesetup.yaml
+  - https://raw.githubusercontent.com/Cimpress-MCP/gitlab-continuous-integration/master/service/dependencyscanning.yaml
+  - https://raw.githubusercontent.com/Cimpress-MCP/gitlab-continuous-integration/master/databasae/builddatabase.yaml
+  - https://raw.githubusercontent.com/Cimpress-MCP/gitlab-continuous-integration/master/databasae/stages.yaml
+
+build:myDataBaseService:
+  extends: .buildDatabaseWithNpm
+  variables:
+    POSTGRES_VERSION: postgres:10.6-alpine
+    POSTGRES_DB: postgres
+    POSTGRES_USER: postgres
+    POSTGRES_SERVER: postgres
+    POSTGRES_PASSWORD: ''
+
+migrate-int:
+  extends: .migrate
+  stage: migrate-int
+  only:
+    - master
+  except:
+    - schedules
+  variables:
+    POSTGRES_SERVER: myDatabaseServerAddress.com
+    POSTGRES_USER: postgres
+    POSTGRES_DB: baton
+    POSTGRES_PASSWORD: $INT_DB_PASSWORD
+
+migrate-prd:
+  extends: .migrate
+  stage: migrate-prd
+  when: manual
+  only:
+    - master
```

Comment:
So, you kinda want this to kick off the prod deploy so we don't have two manual tasks.

We do that by setting `when: on_success` on the deploy-prd task when there is a migrate prod task

https://cimpress.githost.io/commerce/b2b/order-request/blob/master/.gitlab-ci.yml#L61

Which means you need a different deploy prod (or an inherited version with just an override of the `when` field) based on if you have a migrate prod.

Reply:
Eric - this is what I mentioned to you this afternoon about the two manual tasks.

Reply:
I think I have something that should work.